src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -248,6 +248,8 @@ public static IVerbHandler<? extends Reply> responseHandlerOrNoop()
 
     public synchronized static void startup(NodeId tcmId)
     {
+        if (instance != null)
```

Comment:
safety check, I hit an issue and saw we didn't detect this so just added to be safer.